src/components/pages/Table/DataTable.jsx

```diff
@@ -27,10 +41,4 @@ function DataTable({ cases }) {
   );
 }
 
-const mapStateToProps = state => {
```

Comment:
I appreciate you standardizing the coding structure of each file. It makes it easier to follow.